win_images/win_packaging.ps1

```diff
@@ -14,14 +14,24 @@ retryInstall git archiver psexec golang mingw StrawberryPerl; Check-Exit
 # Update service is required for dotnet
 Set-Service -Name wuauserv -StartupType "Manual"; Check-Exit
 
+# Install dotnet
+Invoke-WebRequest -Uri https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.ps1 -OutFile dotnet-install.ps1
+dotnet-install.ps1 -InstallDir 'C:\Program Files\dotnet'
+
+# Configure NuGet sources
+& 'C:\Program Files\dotnet\dotnet.exe' nuget add source https://api.nuget.org/v3/index.json -n nuget.org
+
+# Install wix
+& 'C:Program Files\dotnet\dotnet.exe' tool install --global wix
+
 # dotnet is required for wixtoolset
 # Allowing chocolaty to install dotnet breaks in an entirely
 # non-debuggable way.  Workaround this by installing it as
 # a server-feature first.
-Install-WindowsFeature -Name Net-Framework-Core; Check-Exit
+#Install-WindowsFeature -Name Net-Framework-Core; Check-Exit
 
 # Install wixtoolset for installer build & test.
-retryInstall wixtoolset; Check-Exit
+#retryInstall wixtoolset; Check-Exit
```

Comment:
Assuming what you added above works, my preference would be to just delete all this commented out stuff.  Otherwise anybody else finding this is going to wonder "why" - which isn't commented anywhere.  It's okay to rely on git to remember these commands if we ever need them again.

Reply:
Ack, I'll remove those after testing the built image in the podman windows installer PR. Thanks a lot